### ISSUE TITLE
Police Oversight form: Migrate translated content for thanks form

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1225]

**Comment by desigonz** [comment]
@sergiogcx @amenity I updated translations.json in an attempt to add all missing translations for the thank you form. I submitted [pull request 1225-migrate thank you translations](https://github.com/cityofaustin/officer-thank-form/pull/5). If you merge, I can check it over and see what's missing.

Some questions/comments:

- "Thank the Austin Police Department" --> "Agradezca a APD"—should we translate Austin Police Department?
- Title for the last page is "We have received your thanks" but the translation we have on hand is "¡Nos alegra saber que tuvo una buena experiencia!" I changed to: "Hemos recibido su agradecimiento"

Seems like there are some content discrepancies in the original English content, and I'm wondering if some of the content hasn't been updated. 

- Made new [issue 1325](https://app.zenhub.com/workspaces/office-of-design-and-delivery-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/1325)—office hours are incorrect, the office opens at 8
- On the confirmation page, this doesn't make any sense: "Our job is to make sure your thanks is investigated fairly and thoroughly. Thank you for sharing your experience with us. This helps us better serve you and your community."

**Comment by amenity** [comment]
Thanks, @desigonz! I have a PR out for #1325.

**Comment by desigonz** [comment]
@amenity Thanks for making all of those changes!

I submitted a new [pull request](https://github.com/cityofaustin/officer-thank-form/pull/11) so that translations.json has been updated based on content/field changes in the form.

- I believe our above question about translating Austin Police Department has been mostly solved with the content updates; however, throughout the form we're still using the heading "Thank the APD"—should that instead be "Thank an officer"?
![opo-thanks-APD.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/ba291882-0947-422b-bb21-1e4a7fd81cd8)
